pages/_error.js

```diff
@@ -0,0 +1,70 @@
+// In serverless deployment environments, including Vercel, the Next.js server
```

Comment:
error page that renders when something goes wrong with SSR. just copied this from sentry docs